GUI/KeyView.hs

```diff
@@ -106,10 +106,10 @@ keyData =
      "In the spark conversion  graph the coloured area represents the number " ++
      "of sparks that have fizzled.")
   , ("GCed spark",      KEventAndGraph, gcColour,
-     "As an event it indicates a spark has been GC'd, meaning it has been " ++
```

Comment:
I think these may have been correct, because 'C' is pronounced 'see', so it ends with a vowel.

Reply:
https://github.com/haskell/hie-bios/pull/342/commits/6b6d8777cd41d4004a6fe8869a17f9666a36a4e7

`gced` in above was changed to uppercase

likewise with acknowledgment code (ACK) and process identifier (PID)

less likely to be mistaken in text for a typo (if printed in lowercase)

that being said are the Maintainers amenable to writing it as "GCed"

no apostrophe

https://wiki.haskell.org/GHC/Memory_Management

https://gitlab.haskell.org/ghc/ghc/-/issues/14621

Reply:
I'm not a maintainer, but like the idea of putting GCed everywhere, but not of changing GC'd to GC'ed.

Reply:
removed apostrophe